database/src/main/scala/cromwell/database/liquibase/DiffResultFilter.scala

```diff
@@ -159,5 +171,7 @@ object DiffResultFilter {
       * @return The updated diff result.
       */
     def filterChangedObjects(filters: Seq[DiffFilter]) = filter(diffResult, filters, (_, _) => false)
+
+    def filterOldeObjects = filter(diffResult, Seq.empty, isOldeObject)
```

Comment:
I will `TODO PBE` this as something that should be deleted after the offending Olde tables are deleted.